pkg/proto/aksnodeconfig/v1/tlsbootstrappingconfig.pb.go

```diff
@@ -20,17 +20,78 @@ const (
 	_ = protoimpl.EnforceVersion(protoimpl.MaxVersion - 20)
 )
 
+type BootstrappingAuthMethod int32
+
+const (
+	//nolint:gosec // this is a const string to use in switch statements, not hardcoded credentials
+	BootstrappingAuthMethod_UseTLSBootstrapToken      BootstrappingAuthMethod = 0
```

Comment:
Could we use `= iota` instead of listing all values.

Reply:
you're commenting on generated code we have no control over

btw, I don't `iota` is great for public enums. One day you change order of constants (or remove one of them) and will swap the values in surprising hard-to-track ways.